src/GraphQL/Types/Composite/AutoRegisteringOutputHelper.cs

```diff
@@ -133,7 +133,11 @@ private static bool IsObservableOrAsyncEnumerable(Type type)
     /// that do not return <see langword="void"/> or <see cref="Task"/>
     /// including properties and methods declared on inherited classes.
     /// </summary>
-    public static IEnumerable<MemberInfo> GetRegisteredMembers<TSourceType>(Expression<Func<TSourceType, object?>>[]? excludedProperties)
+#if NET6_0_OR_GREATER
```

Comment:
We could just use `[RequiresUnreferencedCode]` here.  It is not in a public class and warnings should be ignored by the caller, as we have the proper markings on the auto-registering types.